doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                The <a href="http://svnbook.red-bean.com/">Red bean book</a> describes a 
+                version control system.
+                <br/>
+            </div>
+            </p>
+        </ul>
+        <h1><a id="S2">The basics of svn and where to learn more</a></h1>
+        <p>
+        <ul>
+            <div class="page-header">
+                <h3>Files</h3>
+            </div>
+            <div class="well">
+                <li>File sizes</li>
+                Large files and file that are constantly growing can be an issue. 
+                <br/>
+                Files greater than 10MB are not allowed to be committed to the repository 
+                without discussion with the Fcm team.
+                They may be permitted but a quick discussion to assess the need is required.
+                The problem with every growing files is that they are not best suited to a
+                version control repository and may be best located else where.
+                Note they can be in the same directory as your working/local copy but Fcm does 
+                not need to ever know about them.
+                <br/>
+                <br/>
+                <li>File types</li>
+                ASCII v binary files
+                <br/>
+                The main type of files that we will be committing to a repository are: 
+                ASCII and binary files.
+                <br/>
+                <ul>
+                    <li>Assci files can be incrementally updated where only the changed or new 
+                        part is added to the file, for example a txt file, code, LaTeX file.
+                    </li>
+                    <br/>
+                    <li>Binary files are generally larger than ASCII and are only "visible" as a whole 
+                        file and therefore cannot be incrementally updated which means it is often a 
+                        large file which with each new version is saved as a new file and therefore 
+                        the repository can grow very very fast, example files of these files are word 
+                        documents.
+                        These can be saved but it is best to save them once they are complete and 
+                        finalised.
+                        We may wish to write documentation in a text editor or LaTeX to maintain a 
+                        version controlled document.
+                    </li>
+                </ul>
+            </div>
+            <div class="page-header">
+                <h3>Diagrams to explain ideas and concepts</h3>
+            </div>
+            <div class="well">
+                <pre>
+&lt;root&gt;
+    |
+    |-- &lt;project 1&gt;
+    |       |
+    |       |-- trunk
+    |       |-- branches
+    |
+    |-- &lt;project 2&gt;
+    |       |
+    |       |-- trunk
+    |       |-- branches
+    |
+    |-- ...
+
+</pre>
+                The structure of a repository.
```

Comment:
Perhaps a following sentence along the lines of "Notice that a repository contains multiple projects with their own trunk and branches."